sample-apps/react/react-video-demo/src/App.tsx

```diff
@@ -1,5 +1,10 @@
 import { useCallback, useEffect, useState } from 'react';
 import { v1 as uuidv1 } from 'uuid';
+import {
+  MediaPermissionsError,
+  MediaPermissionsErrorType,
+  requestMediaPermissions,
+} from 'mic-check';
```

Comment:
I would avoid installing 3rd party lib, when the browser provides API to check microphone and camera permissions. The `video-react-sdk` has hooks for this (https://github.com/GetStream/stream-video-js/blob/6fac2e53fde35b3c8cc5bbaf26cf8c2d0018127f/packages/react-sdk/src/hooks/useRequestPermission.ts#L5) 

@oliverlaz shouldn't we export those hooks?